<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_morphy">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>形态学处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_charseg">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>字符分割</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_template">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>模板匹配识别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openVideo">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>109</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>提取车牌</string>
    </property>
    <widget class="QLabel" name="lp_img">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>11</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>分割图像</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>10</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lp_seg1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_seg2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_seg3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_seg4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_seg5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_seg6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_seg7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>形态学处理</string>
    </property>
    <widget class="QLabel" name="label_morph">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>车牌号</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>18</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lp_char1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_char2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_char3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_char4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_char5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_char6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lp_char7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>视频</string>
    </property>
    <widget class="QLabel" name="label_video">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>401</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>关键帧</string>
    </property>
    <widget class="QLabel" name="label_keyframe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
